static-content-hosting/README.md

```diff
@@ -0,0 +1,136 @@
+# Static Content Hosting Pattern
+
+This document describes the Static Content Hosting Pattern example from the guide [Cloud Design Patterns](http://aka.ms/Cloud-Design-Patterns).
+
+
+## Before you start
+
+Ensure that you have installed all of the software prerequisites.
+
+The example demonstrates operational aspects of static websites hosted in Azure Storage Accounts with support for static websites. Therefore, you will need to deploy this example in Azure.
+
```

Comment:
Can you just cover all of this in the Prerequisites section instead of having Pre-prerequisites?

```suggestion
```